### PR TITLE
Add cla bot from snowflake-connector-python repo

### PR DESCRIPTION
**Pull request by sfc-gh-rcheng**
Add CLA bot, copied file from other repos:
https://github.com/search?q=org%3Asnowflakedb+CLA_BOT_TOKEN&type=code

### CODE REVIEW

File: .github/workflows/cla_bot.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: "CLA Assistant"
```
Comment (sfc-gh-japatel) [inline .github/workflows/cla_bot.yml:1]: I dont know how to test this but okay to go this in. 

You can potentiall create a new github user and open a PR from fork repo to simulate this.

**Review by sfc-gh-japatel — APPROVED** [review]
lgtm!

File: .github/workflows/cla_bot.yml (hunk 1 above)
Comment (sfc-gh-rcheng) [reply .github/workflows/cla_bot.yml:1]: There is an external PR I'd like to test this on, worst case I'll email them the CLA directly

Comment (sfc-gh-japatel) [reply .github/workflows/cla_bot.yml:1]: once you push this, you can rebase with master and try it out on external PR!

**Pull request merged**